.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+on: push
```

Comment:
```suggestion
on:
  # Run CI against all pushes (direct commits, also merged PRs), Pull Requests
  push:
  pull_request:
```

We want to run on PRs that come from forks too. Push will catch pushes to branches on this project, and merging of PRs, but not the PR itself.

Reply:
Makes sense. Done!